app/repositories/repository_registry.rb

```diff
@@ -27,7 +28,15 @@ def scoped_manual_repository(collection)
         SectionAssociationMarshaller.new(
           section_repository_factory: section_repository_factory,
           decorator: ->(manual, attrs) {
-            DocumentFactoryRegistry.new.manual_with_sections.call(manual, attrs)
+            ManualValidator.new(
```

Comment:
Could you go a step further here and inline this decorator argument into the `SectionAssociationMarshaller`? I don't think the marshaller is ever called with a different value for this decorator.